styles/globals.scss

```diff
@@ -1,4 +1,5 @@
 @import "./settings/colors";
+@import "./settings/spacing";
 @import "./tools/functions";
 
 html,
```

Comment:
@dustin-jw Do we need to keep anything from line 5 to 30?

Reply:
Nah, those can be deleted – if we need some of those styles later, we can put them in a `styles/generic/_reset.scss` file. In general, the entry point should be all `@imports`, no actual CSS.